templates/services_user_data.tpl

```diff
@@ -15,7 +15,7 @@ apt-get update && apt-get -y upgrade
 echo "--------------------------------------------"
 echo "       Setting Private IP"
 echo "--------------------------------------------"
-export PRIVATE_IP="$(/sbin/ifconfig eth0 | grep 'inet addr:' | cut -d: -f2 | awk '{ print $1}')"
+export PRIVATE_IP="$(/sbin/ifconfig ens3 | grep 'inet addr:' | cut -d: -f2 | awk '{ print $1}')"
```

Comment:
does it always end up being in ens3 for these installs?

Reply:
ya as far as i know @GERey

Reply:
or else `enp0s3`: https://askubuntu.com/questions/889880/cannot-find-device-eth0#889882
or `ens33`: https://itzgeek.com/how-tos/mini-howtos/change-default-network-name-ens33-to-old-eth0-on-ubuntu-16-04.html

we should replace this with a shell command to programatically grab the right network interface

Reply:
something like `lshw -C network`